satpy/etc/areas.yaml

```diff
@@ -126,18 +126,6 @@ SouthAmerica_flat:
     lower_left_xy: [-8326322.82790897, -4609377.085697311]
     upper_right_xy: [-556597.4539663679, 1535833.8895192828]
     units: m
-worldeqc30km:
-  description: World in 3km, platecarree
-  projection:
-    proj: eqc
-    ellps: WGS84
-  shape:
-    height: 410
-    width: 820
-  area_extent:
-    lower_left_xy: [-20037508.3428, -10018754.1714]
-    upper_right_xy: [20037508.3428, 10018754.1714]
-    units: m
```

Comment:
Looks like you deleted the world.

Reply:
Yes since it was duplicated, see: https://github.com/pytroll/satpy/blob/a761d23d1e2839b35fa1053a63a4186367c57b17/satpy/etc/areas.yaml#L201-L212

Pycharm was complaining that the key is defined twice, that's why I noticed. There's a small remark in the PR description

Reply:
Ah, so it is.  I missed that you mentioned it in the PR, sorry.

Reply:
No problem :)